bids/layout/layout.py

```diff
@@ -1024,8 +1025,8 @@ def get_metadata(self, path, include_entities=False, scope='all'):
         for layout in self._get_layouts_in_scope(scope):
 
             query = (layout.session.query(Tag)
-                                   .join(BIDSFile)
-                                   .filter(BIDSFile.path == path))
+                     .join(BIDSFile)
+                     .filter(BIDSFile.path == path))
```

Comment:
The old alignment was more readable, IMO.

Reply:
This is a PEP8 warning.    Following guidelines rigorously is usually a practice that I follow as a way to improve overall understanding of code.  That means occasionally making local less appealing formatting.

Personally I have found huge benefits from unconditionally formatting python files with `black`
    
    pip install black
    black --target-version py36 .
    
    Black is the uncompromising Python code formatter. By using it, you
    agree to cede control over minutiae of hand-formatting. In return, Black
    gives you speed, determinism, and freedom from pycodestyle nagging about
    formatting. You will save time and mental energy for more important
    matters.
    
    Blackened code looks the same regardless of the project you're reading.
    Formatting becomes transparent after a while and you can focus on the
    content instead.
    
    Black makes code review faster by producing the smallest diffs possible.
    
    https://github.com/python/black


That's just my 2 cents worth of opinions.  👍  :)